### PR TITLE
Upgrade to rails 5.2.1

### PR DESCRIPTION
**Pull request by msdundar**
- Rails 5.2.1'e yükselt.

### CODE REVIEW

**Comment by greysteil** [comment]
@msdundar would you be up for automatically creating PRs like this one? I built [Dependabot](https://dependabot.com) to do that. Totally free for open source and always will be - I'm just trying to get more people using it so that its [compatibility scores](https://dependabot.com/compatibility-score.html?dependency-name=rails&package-manager=bundler&previous-version=5.2.0&new-version=5.2.1) get super solid.

File: test/config/application_config_test.rb [hunk 1]
```diff
@@ -5,15 +5,18 @@
 # tests config/application.rb configuration to prevent the app from unexpected config changes!
 
 class ApplicationConfigTest < ActiveSupport::TestCase
+  setup do
+    @config = Rails.application.config
+  end
   test 'Configuration defaults should match with Rails version' do
-    assert_equal Rails.application.config.loaded_config_version, Rails.version.to_f
+    assert_equal @config.loaded_config_version, Rails.version.to_f
```
Comment (roktas) [inline test/config/application_config_test.rb]: `to_f` kullanmadan (diğer diff'te göründüğü gibi) Gem::Version'la daha legal yapabilirsin bunu.  Şu çalışmalı (untested):

```ruby
assert_equal Gem::Version.new(@config.loaded_config_version), Gem::Version.new(Rails.version)
```

FYI: Sürüm karşılaştırmalarında başka enteresan case'leri de görmek istersen: https://stackoverflow.com/questions/2051229/how-to-compare-versions-in-ruby

Comment (msdundar) [reply test/config/application_config_test.rb]: Eskiden böyleydi zaten hocam. Bundaki sıkıntı `Gem::Version.new(@config.loaded_config_version)`'un `5.2` döndürüyor olması. Yani verdiğiniz örnek şöyle çalışıyor:

`assert_equal 5.2, 5.2.1`

Bu yüzden `to_f`'ler falan geldi koda :)

**Comment by msdundar** [comment]
@greysteil thanks for notifying us. However, it seems like you don't offer free subscription for open source repositories located under an organization.

**Comment by greysteil** [comment]
@msdundar - they're totally free! Sorry if that wasn't clear - the "Open source / personal account" plan means:
- free for open source, wherever it is (organisation or personal account)
- free for personal account repos, whether they're open source or private.

If you add `omu` under the free plan then you'll be able to use Dependabot on this repo.

**Comment by greysteil** [comment]
Just got an email saying you'd signed up for a "small organisation" account - you definitely don't need that! Looks like it's sorted now and that you're on the free plan, though 🙂.

You shouldn't have been charged for that blip, but if anything goes wrong on GitHub's side and you are then let me know and I'll kick up a stink with GitHub. (I don't control billing GitHub do.)

**Comment by msdundar** [comment]
@greysteil Thanks! Actually I couldn't add organization level repositories by selecting `Open source / personal account`. Switching to `Small Organization` did the trick and allowed me to add repositories. Besides, downgrading back to `Open source / personal account` does no harm after an organizational repository is added.

**Comment by greysteil** [comment]
Huh, I'll take a look into that - sounds like a bug.

Let me know if you ever have any feedback on Dependabot. I took a look at the 3 PRs it generated for you already and they look good to me.

**Pull request merged**